docs/weave/basics/01-blockchain.md

```diff
@@ -10,16 +10,17 @@ A "blockchain" in the simplest sense is a chain of blocks. By chain, we mean eac
 
-This means a blockchain is fully trusted and the state represents the correct function of the programs run on it. This allows for extremely high levels of trust in a program, levels that were previously reserved for highly controlled, centralized systems, such as banks or governments. The first generation of blockchain, Bitcoin, proved it was possible to run a system with many unknown and mutually untrusting parties, yet produce a system that is harder to hack than any bank (bitcoin hacks involve grabbing someone's wallet, not manipulating the blockchain). This was a true marvel of vision and engineering and laid the stage for all future development, and many other projects tried to fork bitcoin to create a custom blockchain.
+This means a blockchain is fully trusted and the state represents the correct function of the programs that run on it. This allows for extremely high levels of trust in a program, levels that were previously reserved for highly controlled, centralized systems, such as banks or governments. The first generation of blockchain, Bitcoin, proved it was possible to run a system with many unknown and mutually untrusting parties, yet produce a system that is harder to hack than any bank (bitcoin hacks involve grabbing someone's wallet, not manipulating the blockchain). This was a true marvel of vision and engineering and laid the stage for all future development, and many other projects tried to fork bitcoin to create a custom blockchain.
 
 ## General Purpose Computer
 
-Ethereum pioneered the second generation of blockchain, where they realized that we didn't have to limit ourselves to handling payments, but actually have a general purpose state machine. They wanted to allow experimentation at a rate orders of magnitude faster than forking bitcoin, and produced the EVM (Ethereum Virtual Machine) that can run sandboxed code uploaded by any user. Since then, hundred of projects have experimented with porting other types of logic to the blockchain, and have demonstrated its utility for [decentralized governance](https://aragon.one/), [currency trading](https://0xproject.com/), [prediction markets](https://gnosis.pm/), even [collectible trading games](https://www.cryptokitties.co/), and much more...
+Ethereum pioneered the second generation of blockchain, where they realized that we didn't have to limit ourselves to handling payments, but actually have a general purpose state machine. They wanted to allow experimentation at orders of magnitude faster than forking bitcoin, and produced the EVM (Ethereum Virtual Machine) that can run sandboxed code uploaded by any user. Since then, hundreds of projects have experimented with porting other types of logic to the blockchain, and have demonstrated its utility for [decentralized governance](https://aragon.one/), [currency trading](https://0xproject.com/), [prediction markets](https://gnosis.pm/), even [collectible trading games](https://www.cryptokitties.co/), and much more...
```

Comment:
```suggestion
Ethereum pioneered the second generation of blockchain, where they realized that we didn't have to limit ourselves to handling payments, but actually have a general-purpose state machine. They wanted to allow experimentation at orders of magnitude faster than forking bitcoin, and produced the EVM (Ethereum Virtual Machine) that can run sandboxed code uploaded by any user. Since then, hundreds of projects have experimented with porting other types of logic to the blockchain and have demonstrated its utility for [decentralized governance](https://aragon.one/), [currency trading](https://0xproject.com/), [prediction markets](https://gnosis.pm/), even [collectible trading games](https://www.cryptokitties.co/), and much more...
```